cl_headers/CMakeLists.txt

```
@@ -4,7 +4,7 @@ if(USE_PREBUILT_LLVM)
 else()
     set(CLANG_COMMAND $<TARGET_FILE:clang>)
 endif()
-if(LLVM_USE_HOST_TOOLS AND NOT OPENCL_CLANG_BUILD_EXTERNAL)
+if(CMAKE_CROSSCOMPILING AND LLVM_USE_HOST_TOOLS AND NOT OPENCL_CLANG_BUILD_EXTERNAL)
```

```suggestion
if(CMAKE_CROSSCOMPILING AND NOT OPENCL_CLANG_BUILD_EXTERNAL)
```
LLVM_USE_HOST_TOOLS is always true if CMAKE_CROSSCOMPILING is true, right?

---

shall we also change LLVM_USE_HOST_TOOLS to CMAKE_CROSSCOMPILING at https://github.com/intel/opencl-clang/blob/9efcfb5f223e5f9fefb854bdb87bcc47655e15ef/CMakeLists.txt#L38

---

> LLVM_USE_HOST_TOOLS is always true if CMAKE_CROSSCOMPILING is true, right?

Yes, thanks -- I think you're right. I had incorrectly assumed LLVM_USE_HOST_TOOLS was a cache variable.

---

> shall we also change LLVM_USE_HOST_TOOLS to CMAKE_CROSSCOMPILING at

Good idea. Done in [6e04828](https://github.com/intel/opencl-clang/pull/464/commits/6e048283f7d61316f5fb55435a39df0f3ace7dc5).